mkosi.extra/usr/lib/tmpfiles.d/etc.conf

```
@@ -9,14 +9,17 @@ L /etc/locale.conf
 L /etc/nsswitch.conf
 L /etc/issue
 L /etc/profile
+# Required by authselect
+L /etc/authselect
 # Required by pam_env plugin
 L /etc/security
 L /etc/bash.bashrc
 L /etc/bash.bash_logout
 # Canonical location to look for certificates
 L /etc/ca-certificates
 L /etc/debuginfod
-L /etc/ssh
+L /etc/ssh/ssh_config
+L /etc/ssh/sshd_config
```

This has the downside, that things that ship includes in e.g. `/etc/ssh/sshd_config.d` will not have them picked upm it might be enough to 
```suggestion
L /etc/ssh/ssh_config
L /etc/ssh/ssh_config.d
L /etc/ssh/sshd_config
L /etc/ssh/sshd_config.d
```